### PR TITLE
Rate limiting and deprecated icons

### PR DESCRIPTION
**Pull request by ProbablePrime**
Rate Limited:

Deprecated:

### CODE REVIEW

**Comment by SimonSchick** [comment]
lgtm

**Comment by UnwrittenFun** [comment]
:+1:

File: src/rest/resource.pug [hunk 1]
```diff
@@ -199,12 +199,24 @@ mixin resourceModals(resource)
 mixin resourceRows(resource)
     if resource.methods
         for method in resource.methods
-            tr.rest-endpoint(onclick=`window.location.href = '#${resource.uniqueId}_${method.method}'`)
+            if method.annotations && method.annotations.deprecated
+                - deprecated = true
+            else
+                - deprecated = false
```
Comment (UnwrittenFun) [inline src/rest/resource.pug]: i don't really know pug but can you not do `deprecated = method.annotations && method.annotations.deprecated` ?

Comment (ProbablePrime) [inline src/rest/resource.pug]: I'll try, pug usually yells at me if i try and be clever

Comment (ProbablePrime) [inline src/rest/resource.pug]: It works \o/

**Pull request merged**